pumps/hybrid.go

```diff
@@ -43,10 +43,10 @@ var (
 
 // HybridPump allows to send analytics to MDCB over RPC
 type HybridPump struct {
-	aggregated             bool
-	trackAllPaths          bool
-	storeAnalyticPerMinute bool
-	ignoreTagPrefixList    []string
+	aggregated               bool
+	trackAllPaths            bool
+	analyticsStoredPerMinute int
```

Comment:
We cannot change configuration fields type - it would break backwards compatibility